definitions.d.ts

```diff
@@ -0,0 +1 @@
+declare module "*.svg";
```

Comment:
this will fix TS errors, but does svgs work correctly in storybook?

Reply:
yes, it works correctly in storybook and I have tested it

Reply:
and if we need to declare modules within our `TS/TSX` file then we can include this file into `tsconfig.types.json` file

Reply:
approved but not sure about the filename, @Weetbix do you have better suggestions?

Reply:
IMO, this is a common pattern for the name of this file as it gives a meaning for what it have also you can find others who name it: `typings.d.ts` or `declarations.d.ts` ...

Reply:
but let's also see what @Weetbix  think about it 😉